chore: Apply style guide (Part 3) (#1151)
SAP/cloud-sdk: docs-js/features/openapi/overview.mdx

### OpenAPI Code Generator

The [OpenAPI code generator](generate-openapi-client.mdx) is a [command line interface (CLI)](generate-openapi-client.mdx#options-of-the-openapi-cli) capable of converting any OpenAPI specification into a TypeScript or JavaScript type-safe client library that is seamlessly integrated with other SAP Cloud SDK features like:

- [connectivity](../connectivity/destination.mdx)
- multi-tenancy
- authentication
- destinations
- and other abstractions of the SAP Business Technology Platform.

It supports OpenAPI versions **2.0** and **3.0**.
You can use the OpenAPI generator to create a client library for any OpenAPI-based API service available on the [SAP API Business Hub](https://api.sap.com/) or found elsewhere in the Internet.

> If you encounter any problems or errors with OpenAPI tooling giving feedback via the [support channels](/docs/overview/get-support).

Reviewer: ```suggestion
If you encounter any problems or errors with the OpenAPI tooling feel free to give us feedback via our [support channels](/docs/overview/get-support).
```

Reviewer: I think this is a rare occasion where "our" is actually appropriate.

## Why the SAP Cloud SDK for JavaScript for OpenAPI Services?

> - Less code boilerplate and better developer experience

Reviewer: ```suggestion
- Less boilerplate code and better developer experience
```

- Simplified development of applications and extensions on the SAP Business Technology Platform
- Convenient abstractions for:
  - Destinations
  - Authentication
  - Service bindings

>   - Automated `CSRF` and `ETag` tokens handling

Reviewer: ```suggestion
  - Automated `CSRF` and `ETag` token handling
```

>   - Automated management of HTTP Headers

Reviewer: ```suggestion
  - Automated management of HTTP headers
```
